content/ja/guides/features/rendering-modes.md

```diff
@@ -1,38 +1,42 @@
 ---
-title: レンダリングモード
-description: レンダリングモード
+title: レンダリング
+description: レンダリング
 position: 1
 category: features
 ---
 
-## ユニバーサル
+## サーバーサイドでレンダリングされるサイトと静的サイト
 
-`mode: 'universal'`: アイソモーフィックなアプリケーション (サーバーサイドレンダリング、または静的なサイト)
+**サーバーサイドでレンダリングされるサイト**は、ユーザーがページをリクエストするたびにサーバー上でレンダリングされるため、リクエストのたびにページを提供できるサーバーが必要です。
+
+**静的サイト**は、サーバーサイドでレンダリングされるアプリケーションと非常によく似ていますが、主な違いとして静的サイトはビルド時にレンダリングされるため、サーバーを必要としません。あるページから別のページへの移動はクライアントサイドで行います。
+
+静的およびサーバーでのホスティングについては [デプロイターゲット](/docs/2.x/features/deployment-targets) を参照してください。
 
 ```js{}[nuxt.config.js]
 export default {
-  mode: 'universal' // デフォルトは universal
+  ssr: true // デフォルトの値
 }
 ```
 
 <base-alert type="info">
-デフォルトのモードはユニバーサルなので、ユニバーサルモードを適用するために nuxt の設定ファイルにこれを追加する必要はありません。
-</base-alert>
 
-## SPA
+デフォルトでサーバサイドレンダリングが有効になっているので、サーバサイドレンダリングを有効にするために `ssr: true` を nuxt の設定に追加する必要はありません。
+
+</base-alert>
 
-`mode: 'spa'`: サーバーサイドレンダリングを行わない (クライアントでのナビゲーションのみ)
+## クライアントサイドレンダリングのみ
 
-`mode` プロパティを利用することで、あなたのプロジェクトのデフォルトの nuxt のモードを変更することができます:
+クライアントサイドレンダリングのみではサーバーサイドレンダリングをすることはできません。クライアントサイドレンダリングとは、JavaScript を使用してブラウザでコンテンツをレンダリングすることです。 HTML からすべてのコンテンツを取得するのではなく、基本的な HTML ドキュメントと JavaScript ファイルを取得し、ブラウザを使用してサイトの残りの部分をレンダリングします。 クライアントサイドレンダリングをするためには ssr を `false` に設定します。
```

Comment:
こちらも同様です

```suggestion
クライアントサイドレンダリングのみではサーバーサイドレンダリングをすることはできません。クライアントサイドレンダリングとは、JavaScript を使用してブラウザでコンテンツをレンダリングすることです。HTML からすべてのコンテンツを取得するのではなく、基本的な HTML ドキュメントと JavaScript ファイルを取得し、ブラウザを使用してサイトの残りの部分をレンダリングします。クライアントサイドレンダリングをするためには ssr を `false` に設定します。
```

Reply:
> こちらも同様に半角スペースは不要です。

こちら a3ee972 で対応いたしました！

> また、原文では Instead of getting all of the content from the HTML... となっているので代わりにを使うのはどうでしょうか？

こちら ac09463 にて対応いたしました！